.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+on:
+  schedule:
+    - cron: "0 9 */5 * *"
+    
+env:
+  SEQ_DB: sequelize_test
+  SEQ_USER: sequelize_test
+  SEQ_PW: sequelize_test
+
+jobs:
+  install-and-build:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16]
+    name: Upload install and build artifact (Node ${{ matrix.node-version }})
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+      - name: Install dependencies and build sequelize
+        run: yarn install --frozen-lockfile
+      - name: Compress artifact
+        run: tar -cf install-build-node-${{ matrix.node-version }}.tar ./lib ./node_modules ./types
+      - uses: actions/upload-artifact@v3
+        with:
+          name: install-build-artifact-node-${{ matrix.node-version }}
+          path: install-build-node-${{ matrix.node-version }}.tar
+          retention-days: 1
+  test-db2-for-i:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16]
```

Comment:
I think we want to run this on 14 as well right?

Reply:
14 & 18 I'd say. Maybe 16 as well if we're willing to let it take that long to run

Reply:
Shall we add 18 in two weeks when the last dependency adds the support for it?

Reply:
Yep